test/distributed/test_c10d.py

```diff
@@ -544,9 +548,13 @@ def _test_store_timeout(self, backend, init_method, c2p):
 
     def _init_methods(self):
         f = tempfile.NamedTemporaryFile(delete=False)
-        yield "file://%s" % f.name
-        f.close()
-        yield "tcp://127.0.0.1:%d" % common.find_free_port()
+        if sys.platform == 'win32':
+            yield "file:///%s" % f.name.replace("\\", "/")
+            f.close()
+        else:
+            yield "file://%s" % f.name
+            f.close()
+            yield "tcp://127.0.0.1:%d" % common.find_free_port()
```

Comment:
Wouldn't that achieve the same goal?
```suggestion
        yield f"{FILE_SCHEMA}{f.name.replace(os.path.sep, '/')}"        
        f.close()
        if sys.platform != 'win32':
            yield "tcp://127.0.0.1:%d" % common.find_free_port()
```